src/printer.ts

```diff
@@ -230,6 +230,13 @@ export class PugPrinter {
 		return !!token && possibilities.includes(token.type) !== invert;
 	}
 
+	private tokenNeedsSeparator(token: AttributeToken): boolean {
+		return (
+			!this.neverUseAttributeSeparator &&
+			(this.alwaysUseAttributeSeparator || /^(\(|\[|:).*/.test(token.name))
```

Comment:
You should run `yarn format` if you are finished with your work

This will become 
```ts
return (
	!this.neverUseAttributeSeparator && (this.alwaysUseAttributeSeparator || /^(\(|\[|:).*/.test(token.name))
);
```

Sadly this is not well readable

So maybe rewrite it to
```ts
if (this.neverUseAttributeSeparator) {
  return false;
}
return this.alwaysUseAttributeSeparator || /^(\(|\[|:).*/.test(token.name));
```

This is longer but increases readability, I think

Reply:
Oh I thought `release.sh` runs `yarn format` also. My bad!

Reply:
Also, maybe `yarn lint` should complain about things that `yarn format` would do differently? BTW, why not just use prettier on the code base? That would reduce a lot of these discussions as there's only one solution 😉

Reply:
Could also be:

```js
		return this.neverUseAttributeSeparator
			? false
			: this.alwaysUseAttributeSeparator || /^(\(|\[|:).*/.test(token.name);
```

If you like ternaries (I do ;) )

Reply:
The one thing with `prettier` is that I format on **save** :laughing:

Reply:
You format on save, or you hate format on save?

Reply:
I format on save / I love it (I even wrote a plugin to format for me on save 😏)